preview/preview.js

```diff
@@ -28,17 +34,44 @@ const updatables = [
     createUpdater("#svg3", {textRotation: 90}),
     createUpdater("#svg4", {textRotation: -90}),
     createUpdater("#svg5", {xAlign: "right"}),
+    createUpdater("#svg6", {
+        textRotation: -90,
+        textShear: 0,
+        xAlign: "right"
+    }),
 ];
 
+// bind text area
 const textArea = document.querySelector("textarea");
-
-function update() {
+function updateText() {
     const text = textArea.value;
-    updatables.forEach(function (updatable) {
-        updatable(text);
+    updatables.forEach((u) => {
+        u.text = text;
+        u.update.apply(u);
+    });
+};
+textArea.addEventListener("change", updateText);
+textArea.addEventListener("keyup", updateText);
+updateText();
+
+// bind shear slider
+const slider = document.querySelector("input#shear");
+function updateShear() {
+    const value = parseInt(slider.value);
+    updatables.forEach((u) => {
+        if (u.options.textShear != null) {
+            u.options.textShear = value;
+            u.update.apply(u);
+        }
     });
 };
+slider.addEventListener("change", updateShear);
```

Comment:
nit: the "input" event will trigger while you're dragging the slider, which could feel a bit better